worker/resume_job_worker.go

```diff
@@ -77,16 +81,26 @@ func (b *ResumeJobWorker) Process(message *workers.Msg) {
 		if err != nil && err == redis.Nil {
 			break
 		}
-		checkErr(b.Logger, err)
+		b.checkErr(job, err)
 		pausedJobArgs, err := workers.NewMsg(batchInfo)
-		checkErr(b.Logger, err)
+		b.checkErr(job, err)
 		pausedJobArr, err := pausedJobArgs.Args().Array()
-		checkErr(b.Logger, err)
+		b.checkErr(job, err)
 		parsed, err := ParseProcessBatchWorkerMessageArray(pausedJobArr)
-		checkErr(b.Logger, err)
+		b.checkErr(job, err)
 		_, err = b.Workers.CreateProcessBatchJob(parsed.JobID.String(), parsed.AppName, &parsed.Users)
-		checkErr(l, err)
+		b.checkErr(job, err)
 	}
 
+	b.Workers.Statsd.Incr(resumeJobWorkerCompleted, job.Labels(), 1)
```

Comment:
Should we not cover this return? https://github.com/topfreegames/marathon/pull/51/files#diff-b8c051ba511ba485573808f5be32c13e02b383b281a59b3ca09cd8b501b5a95fR76